subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/KubernetesReservationClient.kt

```diff
@@ -269,6 +273,11 @@ class KubernetesReservationClient(
             // https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application/#my-pod-stays-waiting
             if (!waitingMessage.isNullOrBlank()) {
                 logger.warn("Can't start pod on this node: $waitingMessage")
+
+                // handle special cases
+                if (waitingMessage.contains("couldn't parse image reference")) {
+                    error("Can't create pods for deployment, check image reference: $waitingMessage")
```

Comment:
Do we have any reference to cause exception?

Reply:
what do you mean with `reference` here?

Reply:
On reference, I mean java object `Throwable`